powersimdata/data_access/csv_store.py

```diff
@@ -50,15 +50,21 @@ def get_table(self):
         :return: (*pandas.DataFrame*) -- the specified table as a data frame.
         """
         filename = self._FILE_NAME
-        try:
-            return self._get_table(filename)
-        except:  # noqa
-            return self._get_table(filename + ".2")
+        orig = self._get_table(filename)
+        blob = self._get_table(filename + ".2")
```

Comment:
Here, we either get a server copy or a local copy in `orig` and always get a blob copy in `blob`, right? Given a blob copy will always be a subset of a server copy/local copy, should we check whether we fail in `orig` (i.e. not connect to server and no local copy) first? And get a blob copy only if `orig` fails?

Reply:
I guess you can a very old copy locally because you lost access to the sever for some reason and meanwhile the blob storage list has been updated with scenarios that are not in the local copy.